extmod/modubinascii.c

```diff
@@ -89,14 +88,89 @@ mp_obj_t mod_binascii_unhexlify(mp_obj_t data) {
 }
 MP_DEFINE_CONST_FUN_OBJ_1(mod_binascii_unhexlify_obj, mod_binascii_unhexlify);
 
+static const char *base64codes = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=";
```

Comment:
That's indeed too "straightforward" implementation, how it's done in some reference code and then copy-pasted to most other implementations around. Did you try to avoid having this "large" static array? I'm sure code can do the same while saving us some bunch of bytes.